### PR TITLE
Add blog post for rubygems adoptions

### PR DESCRIPTION
**Pull request by sonalkr132**
Related: https://github.com/rubygems/rubygems.org/pull/2748

[Rendered](https://medium.com/@sonalkr132/rubygems-adoptions-d09e50afe43)

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby '3.0.1'
+ruby '2.7.4'
```
Comment (jchestershopify) [inline Gemfile:2]: 2.7.4 has a trio of fairly serious CVEs against it. [I'd *strongly* recommend 2.7.5.](https://www.ruby-lang.org/en/news/2021/11/24/ruby-2-7-5-released/)

**Review by jchestershopify — APPROVED** [review]
Apart from the ruby version, LGTM.

File: Gemfile (hunk 1 above)
Comment (jchestershopify) [reply Gemfile:2]: FWIW the same CVEs applied to all previous versions in the 2.6 and 3.0 lines as well, so 3.0.1 is also vulnerable -- 3.0.3 is the patched version in the 3.0 line.

Comment (jchestershopify) [reply Gemfile:2]: Ah, just realised this isn't the .org repo, it's the blog 🤦

Comment (sonalkr132) [reply Gemfile]: thanks. I changed it to 2.7.5. It is related to this https://github.com/rubygems/rubygems.github.io/pull/78#issuecomment-983525018

File: _posts/2021-12-01-rubygems-adoptions.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+All gems with ownership calls will be listed on [a site-wide index page](https://rubygems.org/ownership_calls).
+
+<img src="/images/adoptions/adoptions-02.png" width="100%" />
+
+Any registered user can apply for these ownership calls by creating an ownership request.
+
+<img src="/images/adoptions/adoptions-03.png" width="100%" />
+
+Gem owners will receive a batched email of ownership requests created within the last 24 hours. You can visit the gem adoption page to approve or close ownership requests. If approved, the requester will be added as the owner of the gem. Owners can choose to disable these email notifications.
+Ownership requests
```
Comment (deivid-rodriguez) [inline _posts/2021-12-01-rubygems-adoptions.md:28]: I think this line is not intended?

```suggestion
```

**Review by deivid-rodriguez — APPROVED** [review]
This is really awesome @sonalkr132. Thanks for all you do for the community! ❤️

File: _posts/2021-12-01-rubygems-adoptions.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+At the same time, we are also trying to allow users to request the reuse of an abandoned namespace by contacting the gem owner directly. We routinely receive [support tickets](https://help.rubygems.org/search?utf8=%E2%9C%93&q=abandoned&t=d&scope=all&category_id=&recommend=1) requesting access to an abandoned namespace that hasn't seen a release in quite a while. Finding the contact details of the owners can be a time-consuming process with limited scope for success. Recently we also decided that we will not show [user email publically by default](https://github.com/rubygems/rubygems.org/pull/2663), which users may have used in the past to contact owners. Using "Ownership requests", users will be able to submit such requests to gem owners using rubygems.org.
+
+## Ownership calls
```
[the remark this answers is not in the document]
Comment (sonalkr132) [reply _posts/2021-12-01-rubygems-adoptions.md:14]: Note that "ownership_call" is the name of the model. We are trying to communicate that the user is asking (calling) for new ownerships. I agree that we are trying to invent a term here and this section attempts to explain what "ownership calls" mean.

> any reason to not use more common terminology like "Looking for maintainer"

AFAIU, model names are almost always nouns. I am not sure pluralization would work for "looking_for_maintainer" in the context of the rails association. Further, we are looking for two model names, one created by owners and the other by users. Preferably they should use similar terms. Can you please suggest what could the name for objects created by users in response to "looking for maintainer"?
Are you suggesting we use "looking for maintainer" only for the blog?

> TBH I had no idea what this section is about

Could you please elaborate with examples? I am hoping they are grammatically correct and coherent sentences. It is not very helpful to just say "I have no idea". What changes would you make?

[the remark this answers is not in the document]
Comment (sonalkr132) [reply _posts/2021-12-01-rubygems-adoptions.md:14]: ping

[the remark this answers is not in the document]
Comment (sonalkr132) [reply _posts/2021-12-01-rubygems-adoptions.md:14]: ping

[the remark this answers is not in the document]
Comment (simi) [reply _posts/2021-12-01-rubygems-adoptions.md:14]: Hello, sorry for the delay. We can call that section "Looking for the maintainer" or something and keep the models the same. I think there is no need to keep the model and section names the same.

I think similar problem was raised at https://github.com/rubygems/rubygems.org/issues/2892.

[the remark this answers is not in the document]
Comment (simi) [reply _posts/2021-12-01-rubygems-adoptions.md:14]: > TBH I had no idea what this section is about. 

I was referring to the fact when I open the page, I have no idea what's the purpose thanks to used naming. That's my comment as non English native speaker. Maybe I have just some language gap, that's also possible.

[the remark this answers is not in the document]
Comment (sonalkr132) [reply _posts/2021-12-01-rubygems-adoptions.md:14]: FYI, I have updated the view to address some of your concerns here https://github.com/rubygems/rubygems.org/pull/2910. I am sticking with ownership call/request naming for view with added explanation in subtitle and text area placeholder. I have added "looking for maintainer" as a tooltip. I would appreciate it if you could share your concerns or suggestions before it gets merged.

[the remark this answers is not in the document]
Comment (sonalkr132) [reply _posts/2021-12-01-rubygems-adoptions.md:14]: reminder to review the proposed PR so that we avoid arguments after released.

**Comment by sonalkr132** [comment]
Hi, I am moving ahead with this without confirmation from @simi on whether changes made to the adoption page addressed his concern. I have waited almost a month.

**Comment by simi** [comment]
Sorry :( I had no time to write down my notes.

**Comment by sonalkr132** [comment]
Please create a new issue with any further suggestions you may have.

**Pull request merged**